CMakeLists.txt

```diff
@@ -470,11 +470,37 @@ if(ENABLE_PYTHON)
     message(WARNING "Nettle not found, disabling python support.")
     set(ENABLE_PYTHON OFF CACHE BOOL "" FORCE)
   else()
-    message(STATUS "Python ${Python_VERSION} enabled, using Nettle ${Nettle_VERSION}")
-    target_include_directories(OpenSCAD PRIVATE ${Python_INCLUDE_DIRS})
-    target_link_libraries(OpenSCAD PRIVATE ${Python_LIBRARIES})
+    SET(PYTHON_SOURCES
```

Comment:
Naive question: why not putting (almost) all of this in a `src/python/CMakeLists.txt` file ?